react-app/src/utils/stateName.test.ts

```diff
@@ -0,0 +1,277 @@
+import { describe,  expect, it} from "vitest";
+import {convertStateAbbrToFullName } from "./stateNames";
+
+
+describe('States with Mock inputs', () => {
+    
+    const isStringAState = (state: string, states: Record<string, string>): boolean => {
+        // Check if the state is an abbreviation in the states object
+        if (state in states) return true;
+    
+        // Check if the state matches any full name in the states object
+        return Object.values(states).some((fullName) => fullName.split(", ")[0] === state);
+    };
```

Comment:
This function isn't exposed for anyone to use (it's internal to `convertStateAbbrToFullName`). 

Just calling `convertStateAbbrToFullName('YZ')` should be enough

Reply:
changed the code to just test convertStateAbbrToFullName, since isStringAState is internal to it